libs/dh/wholesale/feature-start/src/lib/dh-wholesale-start.component.ts

```diff
@@ -110,9 +115,34 @@ export class DhWholesaleStartComponent implements OnInit, OnDestroy {
     this.destroy$.complete();
   }
 
+  private getProcessTypes(): void {
+    this.transloco
+      .selectTranslateObject('wholesale.startBatch.processTypes')
+      .pipe(takeUntil(this.destroy$))
+      .subscribe((processTypesTranlation) => {
```

Comment:
Suggestion: The subscribe function can be simplified into this:

`       const keys = Object.keys(ProcessType);
        this.processTypes = keys.map((key) => ({
          displayValue: processTypesTranlation[key],
          value: String(keys.indexOf(key))
        }));`